trunk/SUAVE/Analyses/Aerodynamics/AVL.py

```diff
@@ -116,9 +116,16 @@ def initialize(self):
         self.geometry
         """          
         self.process.compute.lift.inviscid.geometry = self.geometry
+        try:
```

Comment:
The whole point of settings is to default values. If you default the setting then you won't need an if statement.

Reply:
I updated the default settings and removed the if statement